bcl2fastq/lib/bcl2fastq_utils.py

```diff
@@ -297,6 +297,15 @@ def construct_command(self):
         """
         raise NotImplementedError("Subclasses should implement this!")
 
+    def delete_output(self):
+        """
+        Delete the output directory if it exists
+        :return: None
+        """
+        if os.path.isdir(self.config.output):
+            log.info("Found a directory at output path {}, will remove it.".format(self.config.output))
+            shutil.rmtree(self.config.output)
```

Comment:
good point, ties in well with the risk assessment we made about being careful with code that will delete things from disk